Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:alpine
+WORKDIR /src
+RUN apk add --update npm git
+RUN go get -u github.com/jteeuwen/go-bindata/...
+COPY ./webapp/package.json webapp/package.json
```

Comment:
By copying only package.json, then doing the NPM install - we speed up subsequent builds where there the package.json does not change.

npm install is so slow.